src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx

```diff
@@ -21,28 +24,71 @@ export interface AdminSpaceCardProps {
 
 export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
   const backgroundStyle = useCss({
-    "background-image": `url(${venue.mapBackgroundImageUrl})`,
+    background: `linear-gradient(
+        rgba(25, 24, 26, 0.8),
+        rgba(25, 24, 26, 0.8)
+      ), url(${venue.config?.landingPageConfig?.bannerImageUrl})`,
   });
   const backgroundClasses = classNames("AdminSpaceCard__bg", backgroundStyle);
-
   const logoStyle = useCss({
-    "background-image": `url(${venue.host?.icon ?? DEFAULT_VENUE_LOGO})`,
+    "background-image": `url(${venue.host?.icon || DEFAULT_VENUE_LOGO})`,
   });
   const logoClasses = classNames("AdminSpaceCard__logo", logoStyle);
 
+  const spaceIcon = SPACE_PORTALS_ICONS_MAPPING[venue.template];
+
   return (
     <div className="AdminSpaceCard">
-      <div className={backgroundClasses} />
-      <div className="AdminSpaceCard__info">
-        <div className={logoClasses} />
-        <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
+      <div className={backgroundClasses}>
+        <div className="AdminSpaceCard__bg-container">
+          <Link className="AdminSpaceCard__link" to="">
+            {"Visit"}
+            <FontAwesomeIcon
+              className="AdminSpaceCard__link-icon"
+              icon={faExternalLinkAlt}
+            />
+          </Link>
+          <div className="AdminSpaceCard__body">
+            <div className={logoClasses} />
+            <div className="AdminSpaceCard__body-info">
+              <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
+              <span className="AdminSpaceCard__text">
+                <img
+                  alt={`space-icon-${spaceIcon}`}
```

Comment:
Rule of thumb: 

alt text should read in plain English exactly like you'd want some one to explain it to you on the phone.

```suggestion
                  alt={`the ${SPACE_TAXON.lower} icon`}
```

Reply:
done f9043d4